docs/frameworks/fastify.md

```diff
@@ -345,6 +252,17 @@ You can find this URL in the deployment output under the `App Dashboard URL` sec
 
 Your Fastify project will only require the above [adjustments](#deployment-guide), as long as it is written in a **stateless** manner. To find out more about [stateless vs. stateful follow this link](https://stackoverflow.com/questions/5329618/stateless-vs-stateful)
 
+### How to listen on all interfaces?
+
+To listen on all interfaces, you need to set the `host` to `0.0.0.0` and the `port` to the port you want to listen on.
```

Comment:
You can add in the `#troubleshoot` page the log that yields when apps are not listening properly.

It will help users that search for the it on google/docs